source/aks/flux-image-automation.html.md.erb

```diff
@@ -0,0 +1,72 @@
+---
+title: Flux Image Automation
+last_reviewed_on: 2024-02-20
+review_in: 12 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+One of the optional components of flux is image automation. 
+
+This allows you to automatically pick up the latest version of an image and push it to your Flux repo, which is then synced to the cluster.
+
+By doing this, you shouldn't need to manually update a container image version and your cluster should always deploy the latest available version.
+
+To enable this, you set up an image repository and an image policy resource via flux.
+
+An image repository tells flux where it should scan for new images.
+
+
+```
+apiVersion: image.toolkit.fluxcd.io/v1beta2
+kind: ImageRepository
+metadata:
+  name: azure-devops-agent
+  annotations:
+    hmcts.github.com/image-registry: hmctspublic
+spec:
+  image: hmctspublic.azurecr.io/azure-devops-agent
+```
+
+An image policy dictates which newly found images should be applied. By using filters, you can include or exclude certain images based on their tags.
+
+```
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImagePolicy
+metadata:
+  name: default
+spec:
+  filterTags:
+    pattern: '^prod-[a-f0-9]+-(?P<ts>[0-9]+)'
+    extract: '$ts'
+  policy:
+    alphabetical:
+      order: asc
+```
+
+In the example above, only images tagged with something similar to `prod-abcd1234` will be applied.
+
+We have the image automation components set up on our PTL clusters only. 
+
+You can see more info about how the PTL clusters have been configured specifically in the guide on [flux patching](./patching-flux-example.html#prod-environments).
+
+To view the image repos and policies that are on the PTL clusters, you can run:
+
+```
+kubectl get imagerepository.image.toolkit.fluxcd.io -n flux-system
```

Comment:
Want to add examples of the output? May be handy to reference

Reply:
Updated

Reply:
@thomast1906 might reviewing this again? I completely forgot this was open as well